zarf.yaml

```diff
@@ -60,6 +91,12 @@ components:
               fi
             description: "Check k3d version compatibility"
           - cmd: |
+              VOLUME_MOUNT=""
+              if [ "${ZARF_VAR_AIRGAP_MODE}" = "true" ]; then
+                echo "AIRGAP_MODE"
+                export K3D_HELPER_IMAGE_TAG=5.8.3
```

Comment:
We can probably clean up the echo here?

For the env is that needed? If so it should be renovated as well. I could only find one vague reference to it in a k3d issue that noted it's unstable and for dev only, but if it works and we need it that's fine.

Reply:
Yes we need it otherwise it uses whatever version of `k3d` is running locally to determine what helper images to pull. Will renovate